bot/resources/valentines/zodiac_explanation.json

```diff
@@ -0,0 +1,122 @@
+        },
+    "Libra": {
+            "start_at": [9, 23],
+            "end_at": [10, 22],
+            "About": "Amazing people born between **September 23** to **October 22**. Libra is an air sign represented by the scales (interestingly, the only inanimate object of the zodiac), an association that reflects Libra's fixation on balance and harmony. Libra is obsessed with symmetry and strives to create equilibrium in all areas of life.",
+            "Motto": "***“No person is an island.”***",
+            "Strengths": "cooperative, diplomatic, gracious, fair-minded, social.",
+            "Weaknesses": "indecisive, avoids confrontations, will carry a grudge, self-pity.",
+            "full_form": "__**L**__oyal\n__**I**__nquisitive\n__**B**__alanced\n__**R**__esponsible\n__**A**__ltruistic",
+            "url": "https://www.horoscope.com/images-US/signs/profile-libra.png"
+            },
+    "Scorpio": {
+            "start_at": [10, 23],
+            "end_at": [11, 21],
+            "About": "Amazing people born between **October 23** to **November 21**. Scorpio is one of the most misunderstood signs of the zodiac. Because of its incredible passion and power, Scorpio is often mistaken for a fire sign. In fact, Scorpio is a water sign that derives its strength from the psychic, emotional realm.",
+            "Motto": "***“You never know what you are capable of until you try.”***",
+            "Strengths": "resourceful, brave, passionate, stubborn, a true friend.",
+            "Weaknesses": "distrusting, jealous, secretive, violent.",
+            "full_form": "__**S**__eductive\n__**C**__erebral\n__**O**__riginal\n__**R**__eactive\n__**P**__assionate\n__**I**__ntuitive\n__**O**__utstanding",
+            "url": "https://www.horoscope.com/images-US/signs/profile-scorpio.png"
+        },
+    "Sagittarius": {
+            "start_at": [11, 22],
+            "end_at": [12, 21],
+            "About": "Amazing people born between **November 22** to **December 21**. Represented by the archer, Sagittarians are always on a quest for knowledge. The last fire sign of the zodiac, Sagittarius launches its many pursuits like blazing arrows, chasing after geographical, intellectual, and spiritual adventures.",
+            "Motto": "***“Towering genius disdains a beaten path.”***",
+            "Strengths": "generous, idealistic, great sense of humor.",
+            "Weaknesses": "promises more than can deliver, very impatient, will say anything no matter how undiplomatic.",
+            "full_form": "__**S**__eductive\n__**A**__dventurous\n__**G**__rateful\n__**I**__ntelligent\n__**T**__railblazing\n__**T**__enacious adept\n__**A**__dept\n__**R**__esponsible\n__**I**__dealistic\n__**U**__nparalled\n__**S**__ophisticated",
+            "url": "https://www.horoscope.com/images-US/signs/profile-sagittarius.png"
+        },
+    "Capricorn": {
+            "start_at": [12, 22],
+            "end_at": [1, 19],
```

Comment:
here also the ISO format has to be used, but for the ending date, the year needs to be 2021 as we are ending in 2020.